src/extensions/styles/store/reducer.js

```
@@ -22,33 +24,33 @@ function reducer(
-				...state,
-				styles: {
-					...state.styles,
-					...action.styles,
-				},
-			};
+		case 'UPDATE_STYLES': {
+			const chunkSize = 100;
+			const chunks = chunkStylesIntoChunks(action.styles, chunkSize);
+
+			const updatedStyles = chunks.reduce((acc, chunk) => ({
+				...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)